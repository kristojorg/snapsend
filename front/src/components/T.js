import React from 'react';
import Typography from 'material-ui/Typography';

export default ({
  children,
  variant = 'body1',
  ...props
}: {
  variant?: string,
  children: React.Node,
  props: any,
}) => (
  <Typography variant={variant} {...props}>
    {children}
  </Typography>
);
