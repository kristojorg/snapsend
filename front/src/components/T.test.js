import React from 'react';
import T from './T';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

describe('T Component', () => {
  it('T renders correctly', () => {
    const tree = renderer.create(<T>Click me!</T>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('T renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<T>Click me!</T>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});
