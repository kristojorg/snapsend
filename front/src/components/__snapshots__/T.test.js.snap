// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`T Component T renders correctly 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10"
>
  Click me!
</p>
`;
